mypy.ini

```diff
@@ -0,0 +1,24 @@
+[mypy]
+# We'd like to leave follow_imports and ignore_missing_imports at
+# their default values, but there are too many errors to fix for now.
+follow_imports = silent
+ignore_missing_imports = True
+
+# Enable options equivalent to the --strict command line arg
+warn_unused_configs = True
+disallow_subclassing_any = True
+disallow_any_generics = True
+disallow_untyped_calls = True
+disallow_untyped_defs = True
+disallow_incomplete_defs = True
+check_untyped_defs = True
+disallow_untyped_decorators = True
+no_implicit_optional = True
+warn_redundant_casts = True
+warn_unused_ignores = True
+warn_return_any = True
+no_implicit_reexport = True
```

Comment:
why not just use the `--strict` arg? is it better to be explicit in the config?

Reply:
mentioned in an offline conversation, but responding here for posterity: I stuck these here so any editor plugin that knows to run mypy for linting will use the same config we use when running CI.

A side benefit (but also a con) is that our config stays stable if mypy changes what `--strict` does.